components/formats-bsd/src/loci/formats/out/TiffWriter.java

```diff
@@ -80,6 +82,11 @@
   /** TIFF tiles must be of a height and width divisible by 16. */
   private static final int TILE_GRANULARITY = 16;
 
+  /** Automatic tile writing off by default unless set in MetadataOptions using DEFAULT_TILE_WRITING_KEY */
+  public static final String DEFAULT_TILE_WRITING_KEY = "bioformats.tiff.defaultTileWriting";
```

Comment:
See also https://github.com/openmicroscopy/bioformats/pull/2681#discussion_r90903910 for a related discussion. I am also in favor of dropping the `bioformats.` prefix at least when used directly by the API (it makes sense to use a prefix when exposing options in a more complex environment like Fiji/ImageJ where conflicts could happen). Do  we support more than two data layouts? If we ever think of doing it, I would also think something along the lines of `tiff.layout` might be the most flexible option.